awscli/customizations/ec2/secgroupsimplify.py

```diff
@@ -109,8 +109,7 @@ def _add_docs(help_command, **kwargs):
              ' all ICMP types. A value of -1 just for <code>type</code>'
              ' indicates all ICMP codes for the specified ICMP type.</p>')
 CIDR_DOCS = '<p>The CIDR IP range.</p>'
-SOURCEGROUP_DOCS = ('<p>The name or ID of the source security group. '
-                    'Cannot be used when specifying a CIDR IP address.')
```

Comment:
Just to confirm, you *can* specify a source security group as well as a CIDR IP address?